### ISSUE TITLE
¿Cambiar el dns.txt?

### ISSUE THREAD

**Issue by Klint-psk** [romhackhispano/dns#2]
¡Buenas! Tenía una duda. Bueno, dos dudas.
Primera: ¿Cambiar el dns.txt permitiría que la url de klint-psk.romhackhispano.org pudiese apuntar a mi servidor? Más que nada por hacer cambios más fácilmente y que el PHP se ejecute de verdad.
Segunda: En caso afirmativo, ¿cómo hago una "PR" como se especifica? ¿Asumo que es una Pull Request?

**Comment by soywiz** [comment]
No. Tengo que hacerlo desde el proveedor del dominio. Dame la ip del servidor o un cname, y hago que apunte allí :)

**Comment by Klint-psk** [comment]
Pues a ver... he creado un subdominio (klint-psk.jurarigo.com), la IP es 87.98.231.17, pero es compartida por todos los demás subdominios. He hecho un registro CNAME que hace que klint-psk.jurarigo.com se apunte a sí mismo. Creo que con eso debería valer. Creo @_@

**Comment by soywiz** [comment]
La mejor forma de probar si funcionará bien es editar el archivo `hosts` y modificar `klint-psk.romhackhispano.org` para que apunte a `87.98.231.17`. Voy a probarlo, si chuta haré un CNAME al otro subdominio de forma que puedas cambiar ese registro en el caso de que cambies de IP.

**Comment by soywiz** [comment]
Nope, no tienes configurado ese subdominio bien. Al hacer eso sale "Site not installed". Te aconsejo que hagas pruebas con lo que te he comentado, y cuando lo tengas chutando, me avises y lo cambio:

En windows el archivo hosts está en:
`C:\Windows\System32\drivers\etc`

En linux y mac:
`/etc/hosts`

En ambos casos necesitarás privilegios de administrador / root para editarlo.

Simplemente tienes que añadir una línea con `87.98.231.17   klint-psk.romhackhispano.org`, reiniciar tu navegador y meterte en http://klint-psk.romhackhispano.org . Tal y como está ahora te saldrá `Site not installed` y cuando lo arregles, debería salir tu página bien. Y cuando eso ocurra, te cambio el dominio.

**Comment by Klint-psk** [comment]
Lo he probado, pero parece que la IP es la del servidor donde está mi dominio junto a otros muchos, y no la del dominio en sí. He escrito un ticket de soporte para ver si me echan un cable.
Una duda: he mirado el dns.txt y hay varios registros cname que cogen el prefijo de las webs y los mandan aquí a github. De hecho, el mío ahora mismo apunta así. ¿No sería posible mandarlo a una url en lugar de una IP? ¿O solo puede ser para urls dentro del mismo dominio?
